docs/plugins/repository-azure.asciidoc

```diff
@@ -31,7 +32,18 @@ bin/elasticsearch-keystore add azure.client.secondary.account
 bin/elasticsearch-keystore add azure.client.secondary.key
 ----------------------------------------------------------------
 
-`default` is the default account name which will be used by a repository unless you set an explicit one.
+`default` is the default account name which will be used by a repository,
+unless you set an explicit one in the
+<<repository-azure-repository-settings, repository settings>>.
+
+Both `account` and `key` storage settings are
+{ref}/secure-settings.html#reloadable-secure-settings[reloadable]. After the
+reload call returns, all internal azure clients will use the latest
```

Comment:
Unless you follow that "reloadable" link, it's not clear what "the reload call" is.  I recommend something like this: "After you reload the settings, all....".  Ditto in other pages where this text is used.